k8s-readwrite.tf

```diff
@@ -82,6 +82,10 @@ resource "kubernetes_deployment" "apiary_hms_readwrite" {
             name  = "SNS_ARN"
             value = var.enable_metadata_events == "" ? "" : join("", aws_sns_topic.apiary_metadata_events.*.arn)
           }
+          env {
+            name  = "ATLAS_HIVE_SYNC"
+            value = var.enable_atlas_hive_sync == "" ? "" : var.enable_atlas_hive_sync
```

Comment:
What is the value of this variable?  The name makes it sound like a boolean or a `1 / 0` but then you use the value of the TF variable to set the Environment variable, so I am confused.

Reply:
Also, wouldn't you need this in k8s-readonly.tf also?

Reply:
I guess since this is for DDL updates, this doesn't make sense on the RO endpoints, so never mind.

Reply:
Yes I guess I may be just keep it as a boolean, though I dont know if there would be cases where we dont want hive to sync with atlas. I can change the name or value according to what the purpose would be

Reply:
Ok, after looking at https://github.com/ExpediaGroup/apiary-metastore-docker/blob/feature/EGDL-152/files/startup.sh#L87, this makes sense.  I just didn't know what the value of `ATLAS_HIVE_SYNC` was supposed to be - like a class name or something.